src/FormParserFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Parser\FormUrlencoded;
+use React\Http\Parser\Multipart;
+
+class FormParserFactory
+{
+    public static function create(Request $request)
+    {
+        $headers = $request->getHeaders();
+
+        if (!array_key_exists('Content-Type', $headers)) {
+        }
```

Comment:
Left over/Use to return early once I figured out what to return when we can't detect a parser to use.